lib/faker/default/national_health_service.rb

```diff
@@ -6,14 +6,17 @@ class << self
       ##
       # Produces a random British NHS number.
       #
+      # @param use_test_range [Boolean] Whether to use NHS numbers in test range.
+      #   The NHS sets aside a range of numbers from 999 000 0000 to 999 999 9999 for test purposes.
       # @return [String]
       #
       # @example
       #   Faker::NationalHealthService.british_number #=> "403 958 5577"
+      #   Faker::NationalHealthService.british_number(use_test_range: true) #=> "999 464 0232"
       #
       # @faker.version 1.9.2
-      def british_number
-        base_number = rand(400_000_001...499_999_999)
+      def british_number(use_test_range: false)
+        base_number = use_test_range ? rand(999_000_001...999_999_999) : rand(400_000_001...499_999_999)
```

Comment:
What do you think of using a more readable Ruby alternative, i.e.:

```suggestion
        base_number = if use_test_range
          rand(999_000_001...999_999_999)
        else
          rand(400_000_001...499_999_999)
        end
```